politeiawww/user.go

```diff
@@ -182,9 +182,10 @@ func validateUsername(username string) error {
 	return nil
 }
 
-// validatePassword verifies that a password adheres to required policy.
+// validatePassword verifies that a password is of length 64 which is
+// the expected length for sha256 as a string
 func validatePassword(password string) error {
-	if len(password) < www.PolicyMinPasswordLength {
+	if len(password) != 64 {
```

Comment:
Even though this value will never change, we should keep it in a variable to avoid hard coded values throughout the code.